diff --git a/snap.skype.skype.add b/var/lib/snapd/apparmor/profiles/snap.skype.skype
index 33c529a..33dafee 100644
--- a/snap.skype.skype.add
+++ b/var/lib/snapd/apparmor/profiles/snap.skype.skype
@@ -2756,5 +2756,14 @@ userns,
 # dangerous access frivolously.
 deny ptrace (trace),
 deny capability sys_ptrace,
+  #
+  #The following lines added from github.com/AJRepo/apparmor-skype
+  #/sys/devices/LNXSYSTM:00/LNXSYBUS:00/ACPI0003:00/power_supply/AC/online r,
+  #/sys/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C0A:00/power_supply/BAT0/capacity r,
+  /sys/devices/*/*/*/*/*/online r,
+  /sys/devices/*/*/*/power_supply/* r,
+  /sys/devices/*/net/*/speed r,
+  /sys/devices/*/*/net/*/speed r,
+  /etc/issue r,
 
 }

